integration/test_named_vectors.py

```diff
@@ -756,3 +757,48 @@ def test_deprecated_syntax(collection_factory: CollectionFactory):
             return_metadata=wvc.query.MetadataQuery.full(),
         )
     assert "Providing lists of lists has been deprecated" in str(e)
+
+
+@pytest.mark.parametrize(
+    "include_vector, expected",
+    [
+        (False, {}),
+        (["bringYourOwn1"], {"bringYourOwn1": [0, 1, 2]}),
+        # TODO: to be uncommented when https://github.com/weaviate/weaviate/issues/6279 is resolved
+        # (True, {"bringYourOwn1": [0, 1, 2], "bringYourOwn2": [3, 4, 5]})
+    ],
+)
+def test_include_vector_on_references(
+    client_factory: ClientFactory, include_vector: INCLUDE_VECTOR, expected: dict
+) -> None:
```

Comment:
instead of the client factory, could you use the collection factory like the other tests? This way it automatically cleans up before and after. Here are some examples with references: integration/test_collection_references.py

Reply:
Hey @dirkkul , I went with the client factory because i run into errors using the collection factory as you need 2 collections for this test.

Can i still use the collection factory? 
if not can you highlight me the part thats handling the cleaning for the client factory. I can have a look.

Reply:
ok, got it. I am looking at that.

Reply:
replaced client_factory with collection_factory and rerun tests on named_vector.

Reply:
thank you!